### PR TITLE
[IN-22] gomon: fix used memory value

### PR DESCRIPTION
**Pull request by jatinn**
update memory stats: 
linux - take buffer & cache into consideration for available memory
osx - approximate usage to activity monitor usage

### CODE REVIEW

File: plugins/system/collect_linux.go [hunk 1]
```diff
@@ -58,7 +62,8 @@ func (s *System) collect() {
 	}
 
 	if mem_total != 0.0 {
-		s.updaters[memUsageGauge].Update(mem_free / mem_total * 100)
+		avaialbe := (mem_free + buffers + cached)
+		s.updaters[memUsageGauge].Update((mem_total - avaialbe) / mem_total * 100)
```
Comment (jrallison) [inline plugins/system/collect_linux.go]: typo? :)

**Comment by jrallison** [comment]
:+1: other than the available typo

File: plugins/system/collect_linux.go [hunk 2]
```diff
@@ -58,7 +62,8 @@ func (s *System) collect() {
 	}
 
 	if mem_total != 0.0 {
-		s.updaters[memUsageGauge].Update(mem_free / mem_total * 100)
+		avaialbe := (mem_free + buffers + cached)
```
Comment (newhook) [inline plugins/system/collect_linux.go]: typo. avaialbe

Comment (newhook) [inline plugins/system/collect_linux.go]: The commit message should contain the bug number so that it cross references to jira.

**Comment by jatinn** [comment]
fixed typo & commit message

**Pull request merged**